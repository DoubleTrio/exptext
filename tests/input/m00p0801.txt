message_SetFace(ACTOR_NPC_DAGUTORIO, FACE_NORMAL, FACE_POS_STANDARD);
message_Talk({
    english=" Gwah![K] Gwargh![K] Gwagagah! [hero], [c_kind:NPC_ZUBATTO]",
    french=" Gwah![K] Gwaaaah![K]\nGwaaaaahgaaaaah! [hero], [c_kind:NPC_ZUBATTO]",
    german=" Gwah![K] Gwargh![K] Gwagagah! [hero], [c_kind:NPC_ZUBATTO]",
    italian=" Uh![K] Argh![K] Aaaargh! [hero], [c_kind:NPC_ZUBATTO]",
    spanish=" ¡Ah![K] ¡Aaah![K] ¡Aaaaaaaah! [hero], [c_kind:NPC_ZUBATTO]",
});