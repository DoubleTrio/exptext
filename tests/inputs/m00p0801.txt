def 0 {
    with (actor ACTOR_ATTENDANT1) {
        SetAnimation(2);
    }
    with (actor ACTOR_PLAYER) {
        SetAnimation(2);
    }
    with (actor ACTOR_ATTENDANT1) {
        Turn2DirectionLives(4, 10, ACTOR_PLAYER);
    }
    WaitExecuteLives(ACTOR_ATTENDANT1);
    with (actor ACTOR_PLAYER) {
        Turn2DirectionLives(4, 10, ACTOR_ATTENDANT1);
    }
    switch ( ProcessSpecial(PROCESS_SPECIAL_IS_TEAM_SETUP_PARTNER_AND_HERO_ONLY, 0, 0) ) {
        case 1:
            §label_0;
            with (actor ACTOR_ATTENDANT1) {
                Turn2DirectionLives(4, 10, ACTOR_PLAYER);
            }
            WaitExecuteLives(ACTOR_ATTENDANT1);
            message_SetFace(ACTOR_ATTENDANT1, FACE_NORMAL, FACE_POS_STANDARD);
            message_SwitchTalk ($PARTNER_TALK_KIND) {
                case 1:
                    {
                        english=" Hey, [hero].",
                    }
                case 2:
                    {
                        english=" Listen, [hero].",
                    }
                default:
                    {
                        english=" Say, [hero].",
                    }
            }
            CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
            with (actor ACTOR_PLAYER) {
                Turn2DirectionLives(4, 10, ACTOR_ATTENDANT1);
            }
            WaitExecuteLives(ACTOR_PLAYER);
            message_SwitchTalk ($PARTNER_TALK_KIND) {
                case 1:
                    {
                        english=" When we go to [CS:P]Mt. Travail[CR], you\nought to lead, [hero].",
                    }
                case 2:
                    {
                        english=" When we go to [CS:P]Mt. Travail[CR], you\nshould be the leader, [hero].",
                    }
                default:
                    {
                        english=" When we go to [CS:P]Mt. Travail[CR], you\nshould go as the leader, [hero].",
                    }
            }
            message_SwitchTalk ($PARTNER_TALK_KIND) {
                case 1:
                    {
                        english=" After all, that\'s the way we\'ve\nalways done it for important missions.",
                    }
                case 2:
                    {
                        english=" After all, that\'s the way we\'ve\nalways handled things on important missions.",
                    }
                default:
                    {
                        english=" After all, we\'ve always gone on\nour important missions that way.",
                    }
            }
            message_SwitchTalk ($PARTNER_TALK_KIND) {
                case 1:
                    {
                        english=" So I was thinking that you ought\nto be the leader again, [hero]...[K]\nWhat do you say?",
                    }
                case 2:
                    {
                        english=" That\'s why I think you should\nbe the leader again, [hero]...[K]\nWhat do you say?",
                    }
                default:
                    {
                        english=" That\'s why I think you should\nbe the leader again, [hero]...[K]\nWhat do you say?",
                    }
            }
            switch ( message_SwitchMenu(0, 1) ) {
                case menu({
                    english="Accept leadership",
                }):
                default:
                    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
                    switch ( ProcessSpecial(PROCESS_SPECIAL_SET_TEAM_SETUP_HERO_AND_PARTNER_ONLY, 0, 0) ) { }
                    message_SetFace(ACTOR_ATTENDANT1, FACE_NORMAL, FACE_POS_STANDARD);
                    message_SwitchTalk ($PARTNER_TALK_KIND) {
                        case 1:
                            {
                                english=" Cool![K] That\'s it, then! You\'re the\nleader, [hero].",
                            }
                        case 2:
                            {
                                english=" OK![K] That\'s settled. You\'re the\nleader, [hero].",
                            }
                        default:
                            {
                                english=" OK![K] You\'re the team leader,\n[hero]!",
                            }
                    }
                    message_SwitchTalk ($PARTNER_TALK_KIND) {
                        case 1:
                            {
                                english=" Let\'s get rolling to [CS:P]Mt. Travail[CR]!",
                            }
                        case 2:
                            {
                                english=" Let\'s set out for [CS:P]Mt. Travail[CR]!",
                            }
                        default:
                            {
                                english=" We should get moving to\n[CS:P]Mt. Travail[CR]!",
                            }
                    }
                    message_Close();
                    end;
                case menu({
                    english="Refuse",
                }):
                    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
                    se_Play(8968);
                    with (actor ACTOR_ATTENDANT1) {
                        SetEffect(EFFECT_SHOCKED, 3);
                    }
                    with (actor ACTOR_ATTENDANT1) {
                        WaitEffect();
                    }
                    WaitExecuteLives(ACTOR_ATTENDANT1);
                    message_SetFace(ACTOR_ATTENDANT1, FACE_SURPRISED, FACE_POS_STANDARD);
                    message_SwitchTalk ($PARTNER_TALK_KIND) {
                        case 1:
                            {
                                english=" Whoa![K] Hang on, we\'re going\nwith me as leader?",
                            }
                        case 2:
                            {
                                english=" What?[K] Wait, so I\'m going as\nthe leader?",
                            }
                        default:
                            {
                                english=" What?[K] So I\'m going to go as\nthe leader?",
                            }
                    }
                    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
                    message_SetFace(ACTOR_ATTENDANT1, FACE_WORRIED, FACE_POS_STANDARD);
                    message_SwitchTalk ($PARTNER_TALK_KIND) {
                        case 1:
                            {
                                english=" Hmm...[K] I don\'t feel confident\nabout doing that...",
                            }
                        case 2:
                            {
                                english=" Hmm...[K] I don\'t have the\nconfidence to do that...",
                            }
                        default:
                            {
                                english=" Hmm...[K] I don\'t feel confident\nenough to be the leader...",
                            }
                    }
                    message_SwitchTalk ($PARTNER_TALK_KIND) {
                        case 1:
                            {
                                english=" If we\'re going to [CS:P]Mt. Travail[CR],\nyou should be the leader, [hero]...",
                            }
                        case 2:
                            {
                                english=" If we\'re going to [CS:P]Mt. Travail[CR],\nyou should be the leader, [hero]...",
                            }
                        default:
                            {
                                english=" If we\'re still going to go to\n[CS:P]Mt. Travail[CR], you should lead, [hero]...",
                            }
                    }
                    message_Close();
                    end;
            }
            break;
        default:
            message_SetFace(ACTOR_ATTENDANT1, FACE_NORMAL, FACE_POS_STANDARD);
            message_SwitchTalk ($PARTNER_TALK_KIND) {
                case 1:
                    {
                        english=" Hey, [hero].[K] How about if\nwe just go as a pair, you and I?",
                    }
                case 2:
                    {
                        english=" Listen, [hero].[K] What\nwould you say if I suggested that just you and\nI go?",
                    }
                default:
                    {
                        english=" Say, [hero].[K] Do you think\nmaybe we should just go as a pair, us two?",
                    }
            }
            message_SwitchTalk ($PARTNER_TALK_KIND) {
                case 1:
                    {
                        english=" If a whole bunch of us went,\nit\'d probably upset [CS:N]Drowzee[CR] beyond the point\nof persuasion...",
                    }
                case 2:
                    {
                        english=" If a whole group went, [CS:N]Drowzee[CR]\nwould probably get upset. You can forget about\npersuading him then.",
                    }
                default:
                    {
                        english=" If a whole lot of us go, [CS:N]Drowzee[CR]\nwill probably get upset with us. He would be\nimpossible to persuade like that.",
                    }
            }
            message_SwitchTalk ($PARTNER_TALK_KIND) {
                case 1:
                    {
                        english=" But going alone won\'t do it,\neither. One Pokémon won\'t be convincing\nenough.",
                    }
                case 2:
                    {
                        english=" But going alone won\'t be enough.\nI don\'t think it would be persuasive enough to\nget the point across.",
                    }
                default:
                    {
                        english=" But I think one Pokémon alone\nwon\'t be persuasive enough.",
                    }
            }
            message_SwitchTalk ($PARTNER_TALK_KIND) {
                case 1:
                    {
                        english=" So how about just we two go?",
                    }
                case 2:
                    {
                        english=" That\'s why I suggested the two\nof us go.",
                    }
                default:
                    {
                        english=" So what do you think? Should\njust the two of us go?",
                    }
            }
            switch ( message_SwitchMenu(0, 1) ) {
                case menu({
                    english="Go as a pair",
                }):
                default:
                    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
                    switch ( ProcessSpecial(PROCESS_SPECIAL_SET_TEAM_SETUP_HERO_AND_PARTNER_ONLY, 0, 0) ) { }
                    message_SwitchTalk ($PARTNER_TALK_KIND) {
                        case 1:
                            {
                                english=" OK![K] Let\'s get going to\n[CS:P]Mt. Travail[CR]!",
                            }
                        case 2:
                            {
                                english=" That\'s settled![K] Let\'s get going to\n[CS:P]Mt. Travail[CR]!",
                            }
                        default:
                            {
                                english=" OK![K] Let\'s go to [CS:P]Mt. Travail[CR]!",
                            }
                    }
                    message_SwitchTalk ($PARTNER_TALK_KIND) {
                        case 1:
                            {
                                english=" Let\'s do our best, [hero]!",
                            }
                        case 2:
                            {
                                english=" Let\'s do our best, [hero]!",
                            }
                        default:
                            {
                                english=" Let\'s do our best, [hero]!",
                            }
                    }
                    message_Close();
                    end;
                case menu({
                    english="Don\'t go",
                }):
                    message_Close();
                    end;
            }
            break;
    }
}
