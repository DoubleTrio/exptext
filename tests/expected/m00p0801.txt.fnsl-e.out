109     [Kunan](1):  Let\'s get rolling to [CS:P]Mt. Travail[CR]!
34      [Kunan](3):  Say, [hero].
273     [Kunan](3):  Let\'s do our best, [hero]!
67      [Kunan](3):  After all, we\'ve always gone on\nour important missions that way.
269     [Kunan](2):  Let\'s do our best, [hero]!
138     [Kunan](1):  Whoa![K] Hang on, we\'re going\nwith me as leader?
265     [Kunan](1):  Let\'s do our best, [hero]!
26      [Kunan](1):  Hey, [hero].
259     [Kunan](3):  OK![K] Let\'s go to [CS:P]Mt. Travail[CR]!
49      [Kunan](2):  When we go to [CS:P]Mt. Travail[CR], you\nshould be the leader, [hero].
255     [Kunan](2):  That\'s settled![K] Let\'s get going to\n[CS:P]Mt. Travail[CR]!
146     [Kunan](3):  What?[K] So I\'m going to go as\nthe leader?
251     [Kunan](1):  OK![K] Let\'s get going to\n[CS:P]Mt. Travail[CR]!
117     [Kunan](3):  We should get moving to\n[CS:P]Mt. Travail[CR]!
45      [Kunan](1):  When we go to [CS:P]Mt. Travail[CR], you\nought to lead, [hero].
99      [Kunan](2):  OK![K] That\'s settled. You\'re the\nleader, [hero].
238     [Kunan](3):  So what do you think? Should\njust the two of us go?
77      [Kunan](2):  That\'s why I think you should\nbe the leader again, [hero]...[K]\nWhat do you say?
234     [Kunan](2):  That\'s why I suggested the two\nof us go.
59      [Kunan](1):  After all, that\'s the way we\'ve\nalways done it for important missions.
230     [Kunan](1):  So how about just we two go?
162     [Kunan](3):  Hmm...[K] I don\'t feel confident\nenough to be the leader...
224     [Kunan](3):  But I think one Pokémon alone\nwon\'t be persuasive enough.
154     [Kunan](1):  Hmm...[K] I don\'t feel confident\nabout doing that...
220     [Kunan](2):  But going alone won\'t be enough.\nI don\'t think it would be persuasive enough to\nget the point across.
142     [Kunan](2):  What?[K] Wait, so I\'m going as\nthe leader?
216     [Kunan](1):  But going alone won\'t do it,\neither. One Pokémon won\'t be convincing\nenough.
30      [Kunan](2):  Listen, [hero].
210     [Kunan](3):  If a whole lot of us go, [CS:N]Drowzee[CR]\nwill probably get upset with us. He would be\nimpossible to persuade like that.
113     [Kunan](2):  Let\'s set out for [CS:P]Mt. Travail[CR]!
206     [Kunan](2):  If a whole group went, [CS:N]Drowzee[CR]\nwould probably get upset. You can forget about\npersuading him then.
103     [Kunan](3):  OK![K] You\'re the team leader,\n[hero]!
202     [Kunan](1):  If a whole bunch of us went,\nit\'d probably upset [CS:N]Drowzee[CR] beyond the point\nof persuasion...
95      [Kunan](1):  Cool![K] That\'s it, then! You\'re the\nleader, [hero].
196     [Kunan](3):  Say, [hero].[K] Do you think\nmaybe we should just go as a pair, us two?
81      [Kunan](3):  That\'s why I think you should\nbe the leader again, [hero]...[K]\nWhat do you say?
192     [Kunan](2):  Listen, [hero].[K] What\nwould you say if I suggested that just you and\nI go?
73      [Kunan](1):  So I was thinking that you ought\nto be the leader again, [hero]...[K]\nWhat do you say?
188     [Kunan](1):  Hey, [hero].[K] How about if\nwe just go as a pair, you and I?
63      [Kunan](2):  After all, that\'s the way we\'ve\nalways handled things on important missions.
176     [Kunan](3):  If we\'re still going to go to\n[CS:P]Mt. Travail[CR], you should lead, [hero]...
53      [Kunan](3):  When we go to [CS:P]Mt. Travail[CR], you\nshould go as the leader, [hero].
172     [Kunan](2):  If we\'re going to [CS:P]Mt. Travail[CR],\nyou should be the leader, [hero]...
168     [Kunan](1):  If we\'re going to [CS:P]Mt. Travail[CR],\nyou should be the leader, [hero]...
158     [Kunan](2):  Hmm...[K] I don\'t have the\nconfidence to do that...
123     [case_menu]: Refuse
243     [case_menu]: Go as a pair
86      [case_menu]: Accept leadership
279     [case_menu]: Don\'t go
